tests/functional/test_ec2.py

```diff
@@ -34,3 +34,23 @@ def test_delete_tags(self):
         stubber.assert_no_pending_responses()
         self.assertEqual(response, {})
         stubber.deactivate()
+
+
+class TestVolumeDeleteTags(unittest.TestCase):
+    def setUp(self):
+        self.session = boto3.session.Session(region_name='us-west-2')
+        self.service_resource = self.session.resource('ec2')
+        self.volume_resource = self.service_resource.Volume('vol-abc123')
+
+    def test_delete_tags_injected(self):
```

Comment:
Let's move this method and the `setUp()` into a base test class so all you'd have to do is specify the resource in each subclass.